File: morningsnacks/Gcd.java
import java.util.Scanner;
    public class Gcd{
        public static void main(String[] args){

        Scanner input = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int numberOne = input.nextInt();

        System.out.print("Enter another number: ");
        int numberTwo = input.nextInt();

        int gcd = 1;

        for (int count = 1; count <= numberOne && count <= numberTwo; count++){
            if (numberOne % count == 0 && numberTwo % count == 0){
                    gcd = count;
                }    
            }

        System.out.println("GCD is: " + gcd);
        }

    }

File: morningsnacks/gcd.py
number_one = int(input('Enter a number: '))
number_two = int(input('Enter another number: '))

gcd = 1

for count in range(1, number_one +1 and number_two +1):
    if (number_one % count == 0 and number_two % count == 0):
        gcd = count
print('GCD is', gcd)

